source/view/project/project_window.py

```diff
@@ -17,11 +17,7 @@ def __init__(self):
         self.new_action = QAction("New Project")
         self.open_action = QAction("Open...")
         self.save_action = QAction("Save")
-        self.export_png_action = QAction("Image (.png)")
-        self.export_pdf_action = QAction("Image (.pdf)")
-        self.export_tikz_action = QAction("LaTeX (.tikz)")
-        self.export_g6_action = QAction("graph6 (.txt)")
-        self.export_sheet_action = QAction("Sheet (.xlsx)")
```

Comment:
Suggest to rename: 
"graph6 (.txt)" to "graph6 list (.txt)"
"Sheet (.xlsx)" to "Sheet (.xlsx): graph6 and invariants"